SEPADebit/src/main/java/com/braintreepayments/api/SEPADebitRequest.java

```diff
@@ -39,6 +40,7 @@ public String getIban() {
     }
 
     /**
+     * Required.
      * @param iban The full IBAN.
      */
     public void setIban(@NonNull String iban) {
```

Comment:
We wen't with the empty constructor/setter pattern to match most of our other request objects, but all params except merchant account ID are required. Should we add the required params to the constructor instead? Since they are required but technically could be null, should all have `@Nullable` instead of `@NonNull` annotation?

Reply:
I'm up for either, but honestly I do like the empty constructor pattern. Do we callback an exception when a param is missing?

Reply:
I'd say whichever choice makes the API most consistent is the best path to take.

Reply:
Ok will update - I agree leave the empty constructor, I think all the params should be nullable since they _could_ be null, and we'll add checks in the tokenize method to return error if any aren't set 👍

Reply:
Oh wait forgot the API would handle this and return a descriptive error so we left off the null check